renderdoc/driver/vulkan/vk_shader_cache.cpp

```diff
@@ -294,7 +294,7 @@ void VulkanShaderCache::MakeGraphicsPipelineInfo(VkGraphicsPipelineCreateInfo &p
   specMapEntries.resize(specEntries);
   specdata.resize(specSize);
 
-  VkSpecializationMapEntry *entry = &specMapEntries[0];
+  VkSpecializationMapEntry *entry = specEntries ? &specMapEntries[0] : NULL;
```

Comment:
I'd prefer `specEntries.data()` here which amounts to the same thing (not guaranteed to be `NULL`, but we don't need that here as we don't check for `NULL`.